src/ex1/main.js

```diff
@@ -0,0 +1,68 @@
+class Main {
+    constructor() {}
+    init() {
+        const submitButton = domManager.getElement("addTask-btn");
+        submitButton.addEventListener("click", (event) => {
+            let input = domManager.getElement("list-item-input").value;
+            event.preventDefault();
+            // check if number
+            if (/^[0-9]+$/.test(input)) {
+                pokemonClient.getPokemon(input).then((pokemon) => {
+                    let item = pokemon.endsWith("was not found") ?
+                        pokemon // error handling
+                        :
+                        `catch ${pokemon}`; // real pokemon
+                    this.addAndRenderItem(item);
+                });
+            }
+            // check if comma separated list of IDs
+            else if (/^[0-9, ]+$/.test(input)) {
```

Comment:
if you treat a single number as a list with one element - you could always use the getAllPokemons method which will make your code simpler!